InputsTest.java

```diff
@@ -0,0 +1,32 @@
+import org.junit.Test;
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+
+import java.util.List;
+
+public class InputsTest extends BaseTest {
+
+    @Test
+    public void selectInputsTest() {
+        driver.get("http://the-internet.herokuapp.com/inputs");
+        List<WebElement> inputs = driver.findElements(By.tagName("input"));
```

Comment:
There's a single input on the page, why you find it as List<WebElement>?

Reply:
Test corrected